pilot/pkg/model/push_context.go

```diff
@@ -818,6 +818,11 @@ func (ps *PushContext) initAuthnPolicies(env *Environment) error {
 
 	for _, spec := range authNPolicies {
 		policy := spec.Spec.(*authn.Policy)
+		// Fill JwksURI if missing. Ignoring error, as when it happens, jwksURI will be left empty
```

Comment:
This seems reasonable, but /cc @liminw since I think @yangminzhu has made some changes about Jwt Resolving failure handling logic, just want to ensure not regression happens w.r.t networking failure of fetching jwt.

Reply:
The retry logic is inside resolveJwksURIUsingOpenID, which is called by SetAuthenticationPolicyJwksURIs. I think the code change here is fine.